### PR TITLE
Better support for release notes

### PR DESCRIPTION
**Pull request by aviadl**
For any change in content, if it is a modification, you need to add the releaseNotes yml elemnet and describe the change.
If you think that this change shouldnt be logged, add this element with a "-"
Else the build will fail

Also added support for sending mail ask user by user roles

### CODE REVIEW

File: release_notes.py [hunk 1]
```diff
@@ -167,7 +209,15 @@ def addedReleaseNotes(self, cnt):
         return res
 
     def modifiedReleaseNotes(self, cnt):
-        return "- " + cnt["name"] + "\n"
+        rn = cnt.get("releaseNotes", "")
+        if len(rn) == 0:
+            raise Exception(cnt["name"] + "missing release notes yml entry")
+        res = ""
+        #Add a comment only if there are release notes
+        if rn != '-' > 0:
```
Comment (asafshen) [inline release_notes.py]: the `> 0` is redundant...not critical of course...just a bit less readable

File: content-descriptor.json [hunk 2]
```diff
@@ -1,11 +1 @@
...
-  "release": "2.5.0",
-  "id": ""
-}
\ No newline at end of file
+{"releaseDate": "2017-07-04T15:02:58.0+00:00", "release": "2.5.1", "version": -1, "id": "", "installDate": "0001-01-01T00:00:00Z", "assetId": 1234, "releaseNotes": "## Release Notes for version 2.5.1 (1234)\n\n### Integrations\n#### New Integrations\n- MISP\n-- Malware Information Sharing Platform and Threat Sharing\n#### Modified Integrations\n- Cylance Protect\n-- Better error notifications\n- EWS\n-- Support querying multiple mailboxes\n- ServiceNow\n-- Support templates on ticket/incident creation\n\n\n\n\n### Scripts\n#### New Scripts\n- AddEvidence\n-- Adds provided entries to the incident Evidence Board. In playbook, can be positioned after a task to add the previous task's entries to Evidence Board automatically (with no need to provide arguments)\n\n- misp_download_sample\n-- Download malicious file sample from MISP\n- misp_upload_sample\n-- Upload malicious file sample to MISP\n#### Modified Scripts\n- EmailAskUser\n-- Added support to send emails according to roles\n- StringContains\n-- Support looking for one substring out of a list\n", "modified": "2017-07-04T15:02:58.0+00:00", "ignoreGit": false}
```
Comment (asafshen) [inline content-descriptor.json]: `alt` + `cmd` + `L` will prettify this

**Comment by asafshen** [comment]
@aviadl approved - 2 minor notes - for you consideration

**Comment by aviadl** [comment]
@asafshen please check again, fixed your comments

**Comment by asafshen** [comment]
@aviadl still approved :)

**Pull request merged**